app/controllers/my_modules_controller.rb

```diff
@@ -256,6 +256,23 @@ def update
 
   def protocols
     @protocol = @my_module.protocol
+    @recent_protocols = Protocol.where(
+      team: current_team,
+      protocol_type: Protocol.protocol_types[:in_repository_public]
+    ).or(
+      Protocol.where(
```

Comment:
Do we allow loading protocols from other teams?

Reply:
Well I assumed we do, if user has access to them. @radiokills ?